packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -469,7 +469,10 @@ export default class Meetings extends WebexPlugin {
     destroy(meeting, reason) {
       if (this.config.autoUploadLogs) {
         LoggerProxy.logger.info('Meetings->index#destroy Meeting destroyed, uploading logs');
+        const callStart = meeting.locusInfo ? meeting.locusInfo.fullState.lastActive : undefined;
```

Comment:
I see that you are protecting against `locusInfo` not being present. Is there a possibility that `fullState` will always be present if `locusInfo` is present? Otherwise logs might not get uploaded due to that error.

Reply:
Good catch @abailey-wwt we actually always have a `locusInfo` object on the meeting, but the fullstate isn't always there. Will update.

Reply:
Cool. Also we only support latest evergreen browsers correct? Perhaps we can introduce optional chaining? 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
If I remember correctly, the only reason we don't do optional chaining is because the SDK's version of babel doesn't do it. We plan on doing it in SDK v2.